xml/gl.xml

```diff
@@ -11586,7 +11586,8 @@ typedef unsigned int GLhandleARB;
         <enum value="0x95AE" name="GL_SHADING_RATE_SAMPLE_ORDER_DEFAULT_NV"/>
         <enum value="0x95AF" name="GL_SHADING_RATE_SAMPLE_ORDER_PIXEL_MAJOR_NV"/>
         <enum value="0x95B0" name="GL_SHADING_RATE_SAMPLE_ORDER_SAMPLE_MAJOR_NV"/>
-        <unused start="0x95B1" end="0x95B2" vendor="NV"/>
+        <enum value="0x95B1" name="GL_SHADING_RATE_IMAGE_PER_PRIMITIVE_NV"/>
+        <enum value="0x95B2" name="GL_SHADING_RATE_IMAGE_PALETTE_COUNT_NV"/>
```

Comment:
```suggestion
        <enum value="0x95B1" name="GL_SHADING_RATE_IMAGE_PER_PRIMITIVE_NV" group="EnableCap,GetPName"/>
        <enum value="0x95B2" name="GL_SHADING_RATE_IMAGE_PALETTE_COUNT_NV" group="GetPName"/>
```

Reply:
I don't really understand the function of all this, but it sounds like it's a feature that might be used by some third-party tools and not the direct Khronos scripts.  Based on a review of other similar logic, this proposal seems reasonable for the most part.

BUT just to check, if we make this edit, do we also need to add the enums to the corresponding sections referenced by the "group" attribute?  It sounds like maybe not, based on the "Deprecated" comment here?
```
        <group name="EnableCap" comment="Deprecated, use the group attributes instead.">
            <enum name="GL_ALPHA_TEST"/>
            <enum name="GL_ASYNC_DRAW_PIXELS_SGIX"/>
```

Reply:
I tried to use the "Commit suggestion" button here to merge the proposed changes, but I kept getting "This diff has recently been updated.  Refresh and try again."  It never worked, so I pushed an identical commit myself.

Reply:
@nvpbrown you are correct, that stuff is solely for downstream consumers. It should not hold up publication if that's important to you for business reasons.